### ISSUE TITLE
Video thumbnail not being displayed

### ISSUE THREAD

**Issue by everton-schumacker-clansoft** [bakerframework/baker#529]
Hi, 

I am really new in baker framework, so I am still confused with the whole process of building an app with it. I built an standalone app, rendering the pages as three-cards, with media-autoplay set as false, and using html files converted from indesign using in5. Everything seems to work very well, but there are two bugs that I still can't fix.
- The video thumbnails/previews aren't being shown. Everything works fine on browsers, I can see the image preview of the videos in Safari and Chrome, but they are not being displayed on Ipad (not even in the emulator). The video plays normally after you click it. Something that I have notice is that: if you play a video, switch pages and then go back to the previews page, then you can see the thumbnail of the video previously played. It seems like the thumbnail is only loaded to the page after you play the video for the first time.
- The pages are being loaded very slowly, but the real problem is that sometimes it get loaded in the wrong position. All my layouts are 1024X768. Sometimes, the top of the loaded page does not match the top of the screen(it is loaded in a slightly lower position).

Can someone please help with that? Did anyone had this same issue?

Thank you very much!!!

PS: Thank you guys for this framework, it REALLY helped me a lot!!

**Comment by tobiasstrebitzer** [comment]
Bugs related to display and functionality inside an issue are usually within the HPUB document itself.
* The video player is embedded in the HPUB / HTML document, and eventually was not optimized for Mobile Safari (the iOS web browser used by Baker).
* Glitches in positioning are also related to how the HPUB document is built.
A shout-out to In5 should help in both cases.

If you can export a chunk of your document and upload it here, we can have a closer look. 3-4 pages HPUB that includes the misbehaving pages should be enough.

**Comment by everton-schumacker-clansoft** [comment]
Hi, Tobias.

Thanks for your answer! I manage to solve the thumbnail problem. However, the positioning problem is being really hard to track down, specially because it is intermittent: most of the time the page is loaded fine, but sometimes, while changing pages, one or more pages are load in a wrong position. 

I built a sample if you want to have a look, but the book was not exported directly as HPUB. I had to export file by file as html and assemble the book.

[sample.zip](https://github.com/bakerframework/baker/files/134291/sample.zip)
[indesign.zip](https://github.com/bakerframework/baker/files/134292/indesign.zip)
